src/schema/schema.graphql

```diff
@@ -1310,6 +1310,7 @@ type AccountInfoAttrs {
 	zimbraFeatureMobileSyncEnabled: Boolean
 	zimbraFeatureRelatedContactsEnabled: Boolean
 	zimbraFeatureMailForwardingInFiltersEnabled: Boolean
+	zimbraPasswordAllowUsername: String
```

Comment:
should be boolean

Reply:
Yup, Changed it.Also figured out that we have a function coerceStringToBoolean which wasn't being used for resetPassword Response.So, added the same.